trendmicro_deepsecurity/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: trendmicro_deepsecurity
 title: Trend Micro Deep Security
 description: Utilize the Deep Security IPS engine to protect against vulnerabilities without installing a patch (virtual patching)
-version: 1.0.0
+version: 1.1.0
```

Comment:
This will need a major version bump and regeneration because when the connection is modified is requires the user to create a new one.